src/main/webapp/resources/js/apis/pipelines/pipelines.js

```diff
@@ -26,5 +26,10 @@ export async function getIridaWorkflowDescription(workflowUUID) {
  * Get a listing of all Pipelines in IRIDA.
  * @returns {Promise<AxiosResponse<any> | never>}
  */
-export const fetchIridaAnalysisWorkflows = async () =>
-  axios.get(URL).then(response => response.data);
+export const fetchIridaAnalysisWorkflows = async function() {
+  var ajaxUrl = URL;
+  if (window.PAGE.automatedProject != null) {
+    ajaxUrl = ajaxUrl + "?automatedProject=" + window.PAGE.automatedProject;
```

Comment:
Use es6 string literals for stuff like this:
```javascript
ajaxUrl = `${ajaxUrl}?automatedProject=${window.PAGE.automatedProject}`
```

Reply:
Done in 5954ce2